docs/ABI/Mangling.rst

```diff
@@ -176,9 +176,9 @@ Entities
   entity-spec ::= 'fe'                       // ivar initializer; untyped
 
   entity-spec ::= decl-name function-signature generic-signature? 'F'    // function
-  entity-spec ::= decl-name type 'i'                 // subscript ('i'ndex) itself (not the individual accessors)
+  entity-spec ::= type 'i'                           // subscript ('i'ndex) itself (not the individual accessors)
   entity-spec ::= decl-name type 'v'                 // variable
-  entity-spec ::= decl-name type 'f' ACCESSOR
+  entity-spec ::= context entity-spec 'f' ACCESSOR   // enclosed entity is either a variable or subscript
```

Comment:
-) "context" seems to be wrong here. It's already included "entity ::= context ..."

-) We could do a small optimization here: omit the 'f' for 'i' and 'v'. Instead add a pseudo-accessor which is used for plain variables/subscripts, e.g. 
```
variable-spec ::= type 'i' ACCESSOR
variable-spec ::= decl-name type 'v' ACCESSOR
entity-spec ::= variable-spec
...
ACCESSOR ::= 'p' 
```
Accessor symbols are much more frequent than plain variable symbols. For example:
```
...ip  // instead of ...i
...ig  // instead of ...ifg
```